dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -36,12 +40,20 @@ def next_progression_level_or_redirect_path(script_level)
         script_level.next_progression_level
       end
 
-    if script_level.level.try(:plc_evaluation?)
-      enrollment_unit_assignment = Plc::EnrollmentUnitAssignment.find_by(user: current_user, plc_course_unit: script_level.script.plc_course_unit)
-      if enrollment_unit_assignment
-        script_preview_assignments_path
+    if script_level.script.professional_learning_course?
+      if script_level.level.try(:plc_evaluation?)
```

Comment:
Hmm @balderdash may be able to shed some light as to whether this is validated, but there will be [lots](https://github.com/code-dot-org/code-dot-org/blob/40463e7d00504a7b7b9f979aca2332165e3562bb/dashboard/app/helpers/script_levels_helper.rb#L28) [of](https://github.com/code-dot-org/code-dot-org/blob/bcef6440bbeb0eab4ae82d78821874efd6f6187f/dashboard/app/controllers/api_controller.rb#L195) [other](https://github.com/code-dot-org/code-dot-org/blob/bcef6440bbeb0eab4ae82d78821874efd6f6187f/dashboard/app/controllers/api_controller.rb#L243) [places](https://github.com/code-dot-org/code-dot-org/blob/da4607a5f2d033b2331b9da0601737f07f3febfb/dashboard/app/helpers/followers_helper.rb#L14) we'll be sad if `script_level.level` is nil.